module-sys/CMakeLists.txt

```diff
@@ -15,4 +15,5 @@ target_link_libraries(module-sys
         sys-phonemodes
         sys-watchdog
         sys-manager
+        Microsoft.GSL::GSL
```

Comment:
this is `interface` linking here, cant be `private` in this case? I do not really understand why somebody added `interface` linkage in this particular example.
I wouldn't ask for it but you've made it better: [here](https://github.com/mudita/MuditaOS/pull/3925/files#r905896212)

Reply:
ah I see thats my bad on another level. This lib is interface cos it has no own sources - it consists of multiple sys libs. Adding GSL here is a mistake, it shoulld be added one level below in sys-manager lib. Will fix

Reply:
Apart from that, you can't mix interface with private/public specifiers.